pkg/server/router/manifest.go

```diff
@@ -196,33 +208,78 @@ func (mr ManifestRouter) DeleteManifest(ctx context.Context, w http.ResponseWrit
 }
 
 type SubmitApplicationRequest struct {
-	// Once we have JWT signed wrapper that can get the did this can be removed
-	ApplicantDID string                            `json:"applicantDid" validate:"required"`
-	Application  manifestsdk.CredentialApplication `json:"credential_application" validate:"required"`
-	Credentials  []interface{}                     `json:"verifiableCredentials" validate:"required"`
+	ApplicationJWT keyaccess.JWT `json:"applicationJwt" validate:"required"`
+	// Contains the following properties:
+	// Application  manifestsdk.CredentialApplication `json:"credential_application" validate:"required"`
```

Comment:
are we killing the backwards compatibility no signing mode?

Reply:
JWTs all the way down?

Reply:
for requests - yes. responses can return both. we need to accept signed requests.

we can expand this to include data integrity signed payloads too...but not sure what that would look like yet. maybe everything as a VP